tests/puppeteer/campaigns/data/faker/cartRule.js

```diff
@@ -0,0 +1,11 @@
+const faker = require('faker');
+
+module.exports = class Category {
```

Comment:
```suggestion
module.exports = class CartRule  {
```
Am I misundestanding this class ?

Reply:
You're right :) done thank you